### PR TITLE
CVEdit moved and changed coloring of the action buttons

### PR DESCRIPTION
**Pull request by Tiseno**
image

Also changes the color of the green action button as it before looked like a disabled green:
image 
image

Also fixes overflow on the text areas (now hiding the scrollbars (need testing in safari)).

### CODE REVIEW

**Comment by aronstrandberg** [comment]
lol nej på färgerna

**Comment by Tiseno** [comment]
Ja precis, ge mig bättre färger plz 👍 @aronstrandberg

File: app/containers/CvEdit/index.js [hunk 1]
```diff
@@ -144,9 +146,10 @@ export class CvEdit extends React.Component {
     const items =
       section.items.map((item, i) => this.renderItem(item, index, i, section))
     const addItemEnabled = section.items.length < MAX_ITEMS
-    const addItemClasses = addItemEnabled
+    const addItemClasses = classNames(addItemEnabled
       ? 'btn-bright'
-      : 'btn-disabled'
+      : 'btn-disabled', styles.cvAction)
```
Comment (aronstrandberg) [inline app/containers/CvEdit/index.js]: Replacing ternarys with passing an object instead would make it much more readable imo.
e.g:
```
const classes = classNames({
  'btn-bright': addItemEnabled,
  'btn'disabled': !addItemEnabled,
})
```

File: app/containers/App/styles.css [hunk 2]
```diff
@@ -66,11 +66,24 @@ input:focus {
   cursor: pointer;
 }
 
+:global .btn-danger {
+  font-family: 'DIN Condensed';
```
Comment (aronstrandberg) [inline app/containers/App/styles.css]: Let's extract all these common button styles into a global `.btn` style

**Pull request merged**